beater/cache.go

```diff
@@ -0,0 +1,89 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
this file provides rate limiting and not caching, so I'd rename it to something related to rate limiting.